styles/simple.scss

```diff
@@ -1,18 +1,136 @@
 .n-newsletter-signup--simple {
-	display: block;
+	display: flex;
+	flex-direction: row;
+	flex-shrink: 0;
 	border: 0;
+	gap: 20px;
+	padding-top: 20px;
+	background-color: oColorsByName("white-80");
+
+	@include oGridRespondTo($from: S, $until: M) {
+		padding-left: 24px;
+		padding-bottom: 0;
+	}
+
+	@include oGridRespondTo($from: M) {
+		padding-left: 44px;
+		align-items: center;
+		padding-bottom: 20px;
+	}
+
+	.n-newsletter-signup__card_content {
+		display: flex;
+		padding-left: 0;
+
+		@include oGridRespondTo($from: S, $until: M) {
+			flex-direction: column;
+			align-items: start;
+		}
+
+		@include oGridRespondTo($from: M) {
+			flex-direction: row;
+			align-items: center;
+
+		}
+
+		h1 {
+			padding-left: 0;
+		}
+
+		.n-newsletter-signup__body {
+			@include oTypographySans($scale: 0);
+			color: oColorsByName('black-80');
+			display: inline-block;
+
+			@include oGridRespondTo($from: S, $until: M) {
+				width: fit-content;
+			}
+
+			@include oGridRespondTo($from: M) {
+				width: 70%;
+			}
+		}
+	}
+
+	.n-newsletter-signup__top-meta {
+		top: 0;
+		right: 0;
+	}
+
+
+
+	img {
+		max-width: 100%;
+	}
+
+	.newsletter__flex-child-left {
+		position: relative;
+		max-width: 158px;
+		max-height: 89px;
+
+		@include oGridRespondTo($from: M) {
+			overflow: hidden;
+		}
+
+		@include oGridRespondTo($until: M) {
+			flex: 0 0 30%;
+			text-align: center;
+			color: oColorsByName('claret');
+
+		}
```

Comment:
**Change request:** Here we can structure this with a mobile first responsive approach.

It would be something like this 

```suggestion
                flex: 0 0 30%;
		text-align: center;
		color: oColorsByName('claret');

		@include oGridRespondTo($from: M) {
			overflow: hidden;
			// Make any changes you need on M and bigger screens to flex, text-align and color
		}
```